sites/public/src/ListingView.tsx

```diff
@@ -74,13 +72,145 @@ export const ListingView = (props: ListingProps) => {
     unitType: t("t.unitType"),
     rent: t("t.rent"),
     availability: t("t.availability"),
+    ami: t("listings.unit.ami"),
   }
 
   let groupedUnits: Record<string, React.ReactNode>[] = null
-  if (listing.unitsSummary !== undefined && listing.unitsSummary.length > 0) {
-    groupedUnits = getSummariesTableFromUnitsSummary(listing.unitsSummary)
-  } else if (listing.unitsSummarized !== undefined) {
-    groupedUnits = getSummariesTableFromUnitSummary(listing.unitsSummarized.byUnitTypeAndRent)
+  let hmiHeaders: TableHeaders
```

Comment:
I'd love to consider pulling this into sites/public/lib/helpers since this file is already long and hard to parse